tradeexecutor/analysis/trade_analyser.py

```diff
@@ -517,6 +568,47 @@ def show(self):
         with pd.option_context("display.max_row", None):
             df = self.to_dataframe()
             display(df.style.set_table_styles([{'selector': 'thead', 'props': [('display', 'none')]}]))
+    
+    def get_full_report(self):
+        """Show basic and advanced stats and plots"""
+        return (
+            qs.reports.full(self.daily_returns) 
+            if HAS_QUANTSTATS and self.daily_returns is not None
+            else None
+        )
+    
+    def get_basic_stats(self):
+        """Show basic stats only"""
+        return (
+            qs.reports.metrics(self.daily_returns)
+            if HAS_QUANTSTATS and self.daily_returns is not None
+            else None
```

Comment:
Instead of None crash here if HAS_QUANTSTATS is false

```python
if not HAS_QUANTSTATS:
    raise RuntimeError("Quantstats library not installed.")
```